error.go

```diff
@@ -7,10 +7,29 @@ import (
 )
 
 const (
-	releaseNotFoundErrorPrefix = "No such release:"
-	releaseNotFoundErrorSuffix = "not found"
+	cannotReuseReleaseErrorPrefix = "cannot re-use"
 )
 
+var cannotReuseReleaseError = microerror.New("cannot reuse release")
```

Comment:
`"cannot reuse...` vs. `"cannot re-use"` - should these match?

Reply:
This is about the difference of our custom error to have an error and a matcher and having the string matching magic for the untyped upstream errors. Our error here is correct IMO. The matching error string is defined by upstream and it has to be how it is.